scripts/zones/Rolanberry_Fields/mobs/Evil_Weapon.lua

```diff
@@ -0,0 +1,27 @@
+-----------------------------------
+-- Area: Rolanberry Fields
+--  MOB: Evil Weapon
+-----------------------------------
+
+require("scripts/zones/Rolanberry_Fields/MobIDs");
+
+-----------------------------------
+-- onMobDeath
+-----------------------------------
+
+function onMobDeath(mob,killer,ally)
+
+    local mobID = mob:getID();
+    if (Eldritch_Edge_PH[mobID] ~= nil) then
+        local ToD = GetServerVariable("[POP]Eldritch_Edge");
+        if (ToD <= os.time(t) and GetMobAction(Eldritch_Edge) == 0) then
```

Comment:
t is not undefined, its part of `os.time()`'s internal table:
http://www.lua.org/manual/5.1/manual.html#pdf-os.time
`t` tells it "give me the clock in seconds" 
https://github.com/DarkstarProject/darkstar/blob/0307e011f10bb108b6bbc6bf81731ea9f3e7608d/scripts/zones/Korroloka_Tunnel/mobs/Clipper.lua#L20
https://github.com/DarkstarProject/darkstar/blob/0307e011f10bb108b6bbc6bf81731ea9f3e7608d/scripts/zones/Korroloka_Tunnel/mobs/Cargo_Crab_Colin.lua#L14
And is correct here.